resources/edge/config.yaml

```diff
@@ -24,6 +24,14 @@ edge-orders:
 edge-ncip:
   permissions:
   - ncip.all
+edge-dcb:
```

Comment:
change it to edge-dcb: {}

Reply:
Similar to https://github.com/folio-org/pipelines-shared-library/blob/RANCHER-741-Jenkins-Enhancements/resources/edge/config.yaml#L60

Reply:
Done